pkg/admissioncontroller/webhooks/auth/seed/graph/eventhandler_seed.go

```diff
@@ -98,6 +98,9 @@ func (g *graph) handleSeedCreateOrUpdate(seed *gardencorev1beta1.Seed) {
 	namespaceVertex := g.getOrCreateVertex(VertexTypeNamespace, "", gutil.ComputeGardenNamespace(seed.Name))
 	g.addEdge(namespaceVertex, seedVertex)
 
+	leaseVertex := g.getOrCreateVertex(VertexTypeLease, gardencorev1beta1.GardenerSeedLeaseNamespace, seed.Name)
```

Comment:
Should this also be deleted in `handleSeedDelete`?

Reply:
No, there's no need to explicitly handle this in the delete functions (the whole `Seed` vertex will be deleted, and all isolated vertices will go as well. However, the incoming edge should be removed in the beginning of `handleSeedCreateOrUpdate`, ref https://github.com/gardener/gardener/pull/4154#discussion_r647156675

Reply:
Ok, thanks for explaining :)